kairon/shared/catalog_sync/catalog_sync_log_processor.py

```
@@ -0,0 +1,320 @@
+            collection_name=catalog_images_collection,
+            bot=bot,
+            data__image_type="global"
+        ).first()
+
+        if not document:
+            raise Exception(
+                f"Global fallback image document not found in `{catalog_images_collection}`")
+
+        if not document.data.get("image_url"):
+            raise Exception(
+                f"Global fallback image URL not found")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)